rancher/rancher-compose/rancher-services/index.md

```diff
@@ -0,0 +1,186 @@
+
+Request Host | Source Port | Path | Target Port | Label Value
+---|---|---|---| ---
+example.com | 80 | path | 81 | `example.com:80/path=81`
+example.com | 80 | path | | `example.com:80/path`
+example.com | 80||81 | `example.com:80=81`
+example.com |80 | | | `example.com:80`
+example.com | | path | 81 | `example.com/path=81`
+example.com | | path | | `example.com/path`
+example.com| | | 81 | `example.com=81`
+example.com | | | | `example.com`
+ | 80| path | 81 | `80/path=81`
+ | 80 |path | | `80/path` 
+ |80 | | 81 | `80=81`
+ | | path | 81| `/path=81`
+ | | path | | `/path`
+ | | | 81 | `81` *See Note 
+
+<br>
+**Note:** It is assumed that if you have only a port in the label, then the port is for the target port of the service. 
+
+**Load Balancer Sample configuration** `docker-compose.yml`
+
+```yaml
+lb:
+  image: rancher/load-balancer-service
+  ports:
+  # Listen on public port 80 and direct traffic to private port 80 of the service
+  - 80
+  # Listen on public port 82 and by default forward traffic to private port 81 using HTTP
+  - 82:81
+  # Listen on public port 9999 using TCP and not HTTP
+  - 9999/tcp
+  links: 
+  # Any service that is a target will be listed as a link
+  - web1:web1
+  - web2:web2
+  - web3:web3
+  labels:
+    # Put load balancer containers on hosts with label lb=true
+    - "io.rancher.scheduler.affinity:host_label=lb=true"
+    # Requests to http://app.example.com/foo:80 should be routed to web1 over port 8000
+    - "io.rancher.loadbalancer.target.web1=app.example.com:80/foo=8000"
+    # Requests to http://app.example.com should be routed to web2 over port 8000
+    - "io.rancher.loadbalancer.target.web2=app.example.com=8000"
+    # Requests routed to web3 should go to port 8000, overriding the default configuration
+    # of 82:81
+    - "io.rancher.loadbalancer.target.web3=82=8000"
+```
+
+**Sample** `rancher-compose.yml`
+
+```yaml
+lb:
+  # Two load balancer containers
+  scale: 2
+  load_balancer_config:
+    name: lb config
+web1:
+  scale: 1
+web2:
+  scale: 1
+web3: 
+  scale: 1
+```
+
+#### Internal Load Balancer 
+
+To set an internal load balancer, the listening ports are listed under `expose` instead of `ports`.
+
+**Internal Load Balancer Sample configuration** `docker-compose.yml`
+
+```yaml
+lb:
+  image: rancher/load-balancer-service
+  # Instead of using ports, we use expose to define that it will be private ports
+  expose: 
+  # Listen on private port 80 and direct traffic to private port 80 of the service
+  - 80
+  # Listen on private port 82 and by default forward traffic to private port 81 using HTTP
+  - 82:81
+  links: 
+  # Any service that is a target will be listed as a link
+  - web1:web1
+ ```
+
+**Sample** `rancher-compose.yml`
+
+```yaml
+lb:
+  # Two load balancer containers
+  scale: 2
+  load_balancer_config:
+    name: lb config
+web1:
+  scale: 1
+```
+
+
+### External Service
+
+In external services, you can set either external IPs or domain names.
```

Comment:
single domain name, without "s". We should emphasize that both parameters are mutually exclusive.